File: apps/virtual-pet-adoption-server/src/APIs/News/Dtos/NewsWhereInput.cs
namespace VirtualPetAdoption.APIs.Dtos;

public class NewsWhereInput
{
    public string? Content { get; set; }

    public DateTime? CreatedAt { get; set; }

    public string? Id { get; set; }

    public DateTime? PublishDate { get; set; }

    public string? Title { get; set; }

    public DateTime? UpdatedAt { get; set; }
}

File: apps/virtual-pet-adoption-js-admin/src/api/news/NewsWhereInput.ts
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { StringFilter } from "../../util/StringFilter";
import { DateTimeNullableFilter } from "../../util/DateTimeNullableFilter";

export type NewsWhereInput = {
  content?: StringNullableFilter;
  id?: StringFilter;
  publishDate?: DateTimeNullableFilter;
  title?: StringNullableFilter;
};
